recipe/meta.yaml

```diff
@@ -1,12 +1,11 @@
 {% set version = "1.80.0" %}
-{% set filename = "boost_%s.tar.bz2" % version.replace(".", "_") %}
 
 package:
   name: boost-cpp
   version: {{ version }}
 
 source:
-  url: https://boostorg.jfrog.io/artifactory/main/release/{{ version }}/source/{{ filename }}
+  url: https://boostorg.jfrog.io/artifactory/main/release/{{ version }}/source/boost_{{ version|replace(".", "_") }}.tar.bz2
```

Comment:
Should add we know this works well elsewhere ( https://github.com/conda-forge/ace-feedstock/pull/42 )